uwrt_mars_rover_autonomy/uwrt_mars_rover_localization/config/vn300.yaml

```diff
@@ -0,0 +1,43 @@
+# Defined in udev rules, change according to your system. Typically /dev/ttyUSB0
+serial_port: /dev/ttyUSB0
```

Comment:
need to set up udev rules (and commit the rules somewhere too) so that its not random depending on where and when we plug in our usb devices. should be something like /dev/vectornav or smth. see https://github.com/uwrobotics/MarsRover2019/tree/master/udev for some old examples

Reply:
where do we want to commit this?

Reply:
maybe in a new repository? for misc jetson specific stuff? it could have stuff like all our udev rules, maybe instructions on jetson setup, intructions on compiling the kernel with premptrt etc.